include/cute/atom/copy_traits_xe.hpp

```diff
@@ -2198,6 +2367,27 @@ BUILD_XE_NAME(32)
   }
 } // end namespace detail
 
+template <class TiledCopy, class MMA, class MMATensor>
+CUTE_HOST_DEVICE
+auto
+retile_MMA(TiledCopy const&, MMA const&, MMATensor&& mma_tensor) {
```

Comment:
I think this is not used anymore, right?

Reply:
Good catch - removed :+1: